HW1/P2/P2b.py

```diff
@@ -1,3 +1,21 @@
 from P2 import *
+import findspark
+findspark.init()
+import pyspark
+import random
+from random import randint
+sc = pyspark.SparkContext(appName="Spark1")
+x_ = range(2000)
+random.shuffle(x_)
+x = sc.parallelize(x_,10)
+rdd = x.cartesian(x)
+#y = sc.parallelize(xrange(2000),10)
+#rdd = x.cartesian(x).repartition(100)
+#rdd = x.cartesion(x).partitionBy(lambda x:randint(1,101))
+ans = rdd.map(lambda x: [x,mandelbrot((x[0]/500.0-2),(x[1]/500.0-2))])
```

Comment:
You could just use rdd.repartition to randomly assign jobs to different partitions. Your way works, but rdd.repartition is one line!